programs/axelar-solana-gateway/src/executable.rs

```diff
@@ -0,0 +1,407 @@
+//! # Axelar Executable
```

Comment:
I think it would still be nice to have this doc in a separate file/readme so we can have the images rendered on github, etc.

Reply:
Good point. I moved this to [programs/axelar-solana-gateway/axelar-executable.md](https://github.com/eigerco/axelar-amplifier-solana/blob/circular-deps3/programs/axelar-solana-gateway/axelar-executable.md).